.github/workflows/publish-flutter.yml

```diff
@@ -92,6 +92,10 @@ jobs:
           rm android/build.gradle.bak
           rm android/CMakeLists.txt.bak
           rm ios/flutter_breez_liquid.podspec.bak
+      
+      - name: Update version number on podspec files & CMake scripts
+        working-directory: dist
```

Comment:
Is there a reason not to set the working dir here?
```
        working-directory: build/lib/bindings/langs/flutter
```

Reply:
> Is there a reason not to set the working dir here?
> 
> ```
>         working-directory: build/lib/bindings/langs/flutter
> ```

No, fixed: 26b8f83